x-pack/plugins/threat_intelligence/public/modules/indicators/components/indicators_flyout/tabs/indicators_flyout_overview/components/highlighted_values_table/highlighted_values_table.tsx

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useMemo, VFC } from 'react';
+import { Indicator, RawIndicatorFieldId } from '../../../../../../../../../common/types/indicator';
+import { unwrapValue } from '../../../../../../lib/unwrap_value';
+import { IndicatorFieldsTable } from '../../../../components/indicator_fields_table';
+
+/**
+ * Pick indicator fields starting with the indicator type
+ */
+const byIndicatorType = (indicatorType: string) => (field: string) =>
+  field.startsWith(`threat.indicator.${indicatorType}`) ||
+  [
+    'threat.indicator.reference',
+    'threat.indicator.description',
+    'threat.software.alias',
+    'threat.indicator.confidence',
+    'threat.tactic.name',
+    'threat.tactic.reference',
+  ].includes(field);
+
+interface HighlightedValuesTableProps {
+  indicator: Indicator;
+  ['data-test-subj']?: string;
```

Comment:
I like this a lot more than the testId props that I used in my FilterInOut component. I'm taking a mental note to update them to this pattern at some point!